swarm64_run_tpc_benchmark

```diff
@@ -1,15 +1,16 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.6
```

Comment:
Same problem with the Py version. Also, on develop, IMO, the script checks the version already.